src/components/shared/TractionTag.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div :class="[`${getColor}`, 'text-gray-100', 'px-2 rounded-md font-semibold text-sm h-6']">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  /**
+   * # TractionTag
+   *
+   * Tailwind component to display a tag
+   * with a text. Support for two colors
+   *
+   * @example
+   * <template>
+   *   <div>
+   *      <traction-tag>
+   *        Tag Text
+   *      </traction-tag>
+   *   </div>
+   * </template>
+   */
+  name: 'TractionTag',
+  props: {
```

Comment:
This looks almost the same as the traction menu item wondering whether there is any possibility for sharing.

Reply:
I will check on it

Reply:
Traction menu item and Traction tag are actually used for different purposes. 
TractionMenuItem is the top level items displayed in a menu with styles associated with hover and active states. But, TractionTag is more like a static label with a custom style to visualise a highlighted information.
Hope it makes sense